.travis.yml

```diff
@@ -0,0 +1,85 @@
+language: c
+dist: trusty
+sudo: false
+group: beta
+
+# To cache doc-building dependencies.
+cache: pip
+
+os:
+  - linux
+  # macOS builds are disabled as the machines are under-provisioned on Travis,
+  # adding up to an extra hour completing a full CI run.
+  #- osx
+
+compiler:
+  - clang
+  - gcc
+
+env:
+  - TESTING=cpython
+
+matrix:
+  allow_failures:
+    - env:
+       - TESTING=coverage
+  include:
+    - os: linux
+      language: python
+      python: 3.5
+      env:
+        - TESTING=docs
+      before_script:
+        - cd Doc
+        - make venv PYTHON=python3
+      script:
+        - make html SPHINXBUILD="./venv/bin/python3 -m sphinx" SPHINXOPTS="-q"
+    - os: linux
+      language: c
+      compiler: clang
+      env:
+        - TESTING=coverage
+      before_script:
+        - ./configure
+        - make -s -j4
+        # Need a venv that can parse covered code.
+        - ./python -m venv venv
```

Comment:
This won't fly with 2.7, but I'm not sure what we can do instead to fix it.  On the other hand, do we care as much about coverage on 2.7?